cdc/owner/ddl_sink.go

```diff
@@ -426,7 +427,13 @@ func (s *ddlSinkImpl) close(ctx context.Context) (err error) {
 
 // addSpecialComment translate tidb feature to comment
 func (s *ddlSinkImpl) addSpecialComment(ddl *model.DDLEvent) (string, error) {
-	stms, _, err := parser.New().Parse(ddl.Query, ddl.Charset, ddl.Collate)
+	p := parser.New()
+	mode, err := mysql.GetSQLMode(s.info.Config.SQLMode)
```

Comment:
Can you please give more comment for the SQLMode here, which type means can handle "" quoted names?

Reply:
Besides "" quoted names, since we introduced SQLMode here, what other cases we can handle for other SQLMode types?

Reply:
I will add more detail in the TiCDC documentation.